app/actions/CommonActions.scala

```diff
@@ -56,12 +56,15 @@ object CommonActions {
   implicit class MobileSupportRequest[A](val request: Request[A]) extends AnyVal {
 
     private def _platform = request.getQueryString("platform").orElse(request.session.get("platform")).getOrElse("web")
+    private def _sessionId = request.cookies.get("gu.contributions.csrf-token").map(_.value).getOrElse("Unknown")
```

Comment:
It would be good not to couple this to the CSRF cookie (which has an uncertain future). I think the Play session would be the place to do it – storing a UUID there should do the trick. https://www.playframework.com/documentation/2.6.x/JavaSessionFlash#Storing-data-into-the-Session